optout/src/main/java/gov/cms/ab2d/optout/OptoutJob.java

```diff
@@ -0,0 +1,25 @@
+package gov.cms.ab2d.optout;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+
+@Slf4j
+@RequiredArgsConstructor
+public class OptoutJob extends QuartzJobBean {
+
+    private static int runCount = 0;
+
+    private final OptOutProcessor processor;
+
+    @Override
+    protected void executeInternal(JobExecutionContext context) throws JobExecutionException {
+        log.info("##########################################################################");
```

Comment:
Let's remove this logging statement to avoid sending unnecessary stuff to Splunk.

Reply:
done